net/redisclient.go

```diff
@@ -87,6 +101,96 @@ const (
 	defaultConnMetricsInterval = 60 * time.Second
 )
 
+// JumpHash
+// https://arxiv.org/pdf/1406.2294.pdf
+type JumpHash struct {
```

Comment:
Should this be private `jumpHash`?

Reply:
It could be reused, but I was not sure about the others. I am happy to demote to private

Reply:
The original jumphash library could be utilized if reuse is required

Reply:
done